Source/WebCore/css/parser/CSSPropertyParser.h

```diff
@@ -156,6 +156,8 @@ class CSSPropertyParser {
     bool consumeTextWrapShorthand(bool important);
     bool consumeWhiteSpaceShorthand(bool important);
 
+    bool consumeAnimationRangeShorthand(bool important);
```

Comment:
This class seems disorganized, with no obvious logic about what order and grouping there is for the function declarations. Not a new issue though. I would probably sort them alphabetically.

Reply:
Moved this to be with the other scroll driven animation consumers.